node/os.ts

```diff
@@ -136,7 +136,15 @@ export function getPriority(pid = 0): number {
 
 /** Returns the string path of the current user's home directory. */
 export function homedir(): string | null {
-  notImplemented(SEE_GITHUB_ISSUE);
+  switch (Deno.build.os) {
+    case "windows":
+      return Deno.env.get("HOMEPATH") || null;
```

Comment:
I think 'USERPROFILE' is more reliable env var for pointing the home directory in windows. See this discussion for example https://github.com/denoland/deno/pull/3462

Node.js documentation also says it uses 'USERPROFILE' https://nodejs.org/api/os.html#os_os_homedir